project_cards/year_2015_antioch_bridge_extension.yml

```diff
@@ -0,0 +1,42 @@
+project: year 2015 antioch bridge extension
+tags:
+- year 2015
```

Comment:
add a `highway review` tag

Reply:
Was waiting on notebook to run to commit,.

committed now, notebook is running